libs/designsystem/src/lib/components/icon/kirby-icon-registry.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+
+import { Icon } from './icon-settings';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class KirbyIconRegistryService {
```

Comment:
Consider naming - we don't use the `Kirby` prefix in general, as it's already exported to / imported from `@kirbydesign/designsystem`
```suggestion
export class IconRegistryService {
```

Reply:
Might be beneficial here. Look how Angular Material prefixes their registry service with eg. MatIconRegistry.

https://material.angular.io/components/icon/api

Reply:
An also its more explicit that its a kirby service, without having to look at the package import.

Reply:
That might be, but consistency is of importance here, so please keep naming consistent :)

Reply:
Valid point if kirby prefix not used elsewhere!